Moved git1 and git2 into their own directories

adamblanchard opened the pull request:
Since we teach git1 and git2 separately, we updated the directory structure to reflect that (similar to how we do it for javascript modules)

IslamFawzy25 commented:
LGTM

ahmagdy commented:
I suggest renaming week1 to git101 and week2 to git102.
Using git1/week1 is redundant routing since we only have one week per module

adamblanchard commented:
@ahmagdy my intention was consistency with how we structure the other modules that are split up (e.g. javascript). Totally fair that it feels redundant to have weekX in there though.

How about we just have Git1 and Git2 directories by themselves? Then it reflects what we call it in the overall curriculum still but has no redundant week directories

ahmagdy commented:
> @ahmagdy my intention was consistency with how we structure the other modules that are split up (e.g. javascript). Totally fair that it feels redundant to have weekX in there though.
> 
> How about we just have Git1 and Git2 directories by themselves? Then it reflects what we call it in the overall curriculum still but has no redundant week directories

Yup, this sounds like a fine idea :)

adamblanchard commented:
@ahmagdy I've updated it as described, and also updated the readme (which i missed last time) to update the links and description to reflect this new directory structure.

Can I get an approval (unless there is anything else :D)?

(Or just a thumbs up if you don't have permission 🤔 )

ahmagdy reviewed (approved):
Thank you for the amazing work :)
